### ISSUE TITLE
Cannot create a server workspace

### ISSUE THREAD

**Issue by julia-vaseva** [microsoft/azure-devops-intellij#91]
I have a question concerning server workspaces in Team Services plugin. As I have gathered from [this issue](https://github.com/Microsoft/vso-intellij/issues/47), by default a local workspace is created, but there are some personal configurations, which allow creating a server workspace. Would you please explain what these configurations are? For me, a workspace is always local and "Location" dropdown is always disabled.
![image](https://user-images.githubusercontent.com/30112159/28830824-9627c3fc-76e0-11e7-94a1-3eb98146ead6.png)

**Comment by leantk** [comment]
Hi @julia-vaseva,

We currently only support local workspaces. In the next release of the plugin that will come out this week (version 1.122.0) we will allow the option to edit server workspaces that already exist but no other operations at this time (essentially the menu you showed above will work for server workspaces). The dropdown is disabled because we do not want users to change their workspaces to server workspaces and then not be able to use them with the plugin since we don't support them yet. 

The configurations for creating a server workspace would be if the user created the workspace through the TFS plugin (not our plugin) or from the command line tool. We force all workspaces created through our plugin to be local. Does that make sense?

Thanks,
-Leah

**Comment by julia-vaseva** [comment]
Hi @leantk,

Thanks for the explanation. I wonder if there are any plans to add full support for server workspaces? If yes, could we expect it soon?

I suppose that creating a local workspace by default is ok since there is an option to create a server one via console. Anyway, I will let you know if I receive any requests about server workspace support.

Julia

**Comment by leantk** [comment]
We do have plans to eventually support server workspaces but I don't have a definitive timeline on when. Local workspaces are the more commonly used of the 2 so that's why we went with supporting them first and then planned to go back to support server workspaces. I'll keep this issue open and track our progress with this work here.

Thanks,
-Leah

**Comment by vladad** [comment]
Dear @leantk,
are there any new information about server workspaces support for TFS 2015+?

In [IntelliJ TFS Integration plugin](https://plugins.jetbrains.com/plugin/4578-tfs-integration) for exapmle checking out existing files or adding new files works perfectly with TFS 2017, but local changes are not visible in IDE.
I have created new issue [them](https://youtrack.jetbrains.com/issue/IDEA-188491) but they said, that they support up to TFS 2015 and no TFS 2017.

With yours plugin I tried some hacks in `TEE-CLC-14.123.1\tf.cmd` with specifying additional arguments  `-server:"http://host:8080/tfs/collection" -login:"domain\name,password"` for TF process executing but still some authorization or other errors occurs.
